.env

```
@@ -18,6 +18,7 @@ NGINX_VERSION=1.25
 
 # Frontend configuration
 NODE_VERSION=22
+APP_URL=http://localhost:8080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_URL key should go before the NODE_VERSION key

(UnorderedKey)